File: lab4/01_data_load.py
# Create a duckdb database and save penguins df as a table in it (alternative to 01_data_load.R)
import duckdb
from palmerpenguins import penguins

con = duckdb.connect('my-db.duckdb')
df = penguins.load_penguins()
con.execute('CREATE TABLE penguins AS SELECT * FROM df')
con.close()

File: lab4/01_data_load.R
# Create a duckdb database and save penguins df as a table in it (alternative to 01_data_load.py)
con <- DBI::dbConnect(duckdb::duckdb(), dbdir = "my-db.duckdb")
DBI::dbWriteTable(con, "penguins", palmerpenguins::penguins)
DBI::dbDisconnect(con)
